scripts/generate_api_docs.py

```diff
@@ -52,6 +52,7 @@
     Path("aea", "decision_maker", "scaffold.py"),
     Path("aea", "error_handler", "scaffold.py"),
     Path("aea", "test_tools", "click_testing.py"),
+    Path("aea", "protocols", "dialogue", "base.py"),
```

Comment:
That's not what I mean - I meant ignoring specific errors so we can keep the file